src/aspire/storage/starfile.py

```diff
@@ -115,11 +113,13 @@ def _initialize_blocks(self):
                     )
             elif block_has_loop:
                 if gemmi_block.name not in self.blocks:
-                    # initialize DF from list of lists
-                    # read in with dtype=str because we do not want type conversion
-                    self.blocks[gemmi_block.name] = pd.DataFrame(
-                        loop_data, columns=loop_tags, dtype=str
-                    )
+                    # initialize dict from list of lists
+                    d = {}
+                    for i, loop_tag in enumerate(loop_tags):
+                        d[loop_tag] = [
+                            str(loop_data[j][i]) for j in range(len(loop_data))
```

Comment:
`[str(x[i]) for x in loop_data]`?

Reply:
yeah, sorry I missed that one too, replaced

Reply:
Thanks!